collect_app/src/main/java/org/odk/collect/android/views/SlidingTabLayout.java

```diff
@@ -203,7 +204,14 @@ private void populateTabStrip() {
             }
 
             if (mTitleFontColor != -1) {
-                tabTitleView.setTextColor(getResources().getColor(mTitleFontColor));
+
+                int tabTitleTextColor;
```

Comment:
This doesn't help because you set the theme to null! There is a better way to address this deprecation that will actually take into account the theme.

Also make sure that you have spaces around operators and after } and ,.